src/Components/Authentication/__tests__/Desktop/ModalManager.test.tsx

```diff
@@ -57,4 +57,36 @@ describe("ModalManager", () => {
 
     expect(document.body.style.overflowY).toEqual("hidden")
   })
+
+  it("handles type changes", () => {
+    const wrapper = getWrapper()
+    let manager = wrapper.instance() as ModalManager
+
+    manager.openModal({
+      mode: "login",
+    })
+
+    manager.handleTypeChange("signup")
+
+    expect(manager.state.currentType).toEqual("signup")
+    expect(manager.state.switchedForms).toEqual(true)
+    expect(manager.state.options.mode).toEqual("signup")
+  })
+
+  it("returns the right subtitle", () => {
+    const wrapper = getWrapper()
+    let manager = wrapper.instance() as ModalManager
+
+    manager.openModal({
+      mode: "login",
+      copy: "Foobar",
```

Comment:
Are we considering changing `copy` to `subTitle`?

Reply:
Yep we've considered, but probably for a separate PR. We'll need to carve time to refactor this since `copy` is used all over the place in Force.